stripe/_api_requestor.py

```diff
@@ -538,6 +617,90 @@ def request_raw(
 
         return rcontent, rcode, rheaders
 
+    async def request_raw_async(
```

Comment:
nit: remove?

Reply:
No, `request_raw` is an internal helper function for `request`, it is not `raw_request`, and we need the asynchronous version.

Reply:
We should consider renaming